docs/SDK/Explanation/authentication/sessionSigs/authMethods/email-sms.md

```diff
@@ -0,0 +1,112 @@
+# Email / SMS
+
+When authenticating with email or sms provided otp codes, it is a two step process. First initating a code be sent to a email / phone number, then asserting on the user provided code which will generated a signed Json Web Token if successful. This token will be validated when signing Session Signatures as the `accessToken` within the returned `Auth Method`.
+
+:::note
+Codes sent to users via email will be recieved from `noreply@litprotocol.com`. Codes sent to users via sms will include `lit-verification` within the sms message.
+:::
+### Register user with sms / email auth
+
+```javascript
+const authClient = new LitAuthClient({
+    litRelayConfig: {
+        relayApiKey: 'your-api-key',
```

Comment:
for consistent styling and making these filler values more distinct,

```suggestion
        relayApiKey: '<Your Lit Relay Server API Key>',
```

check out https://github.com/LIT-Protocol/docs/blob/main/docs/SDK/Explanation/authentication/sessionSigs/authMethods/socialLogin.md as an example